packages/frontend/src/pages/CreateProject.tsx

```diff
@@ -24,39 +26,63 @@ const CreateProject: FC = () => {
     const [selectedWarehouse, setSelectedWarehouse] = useState<
         SelectedWarehouse | undefined
     >();
+    const [hasDimensions, setHasDimensions] = useState<string | undefined>();
```

Comment:
These new panels should only show on the first time. We can use the `orgData.needsProject` to know if it is the first project.
```suggestion
    const [hasDimensions, setHasDimensions] = useState<string | undefined>(orgData.needsProject ? 'hasDimensions' : undefined);
```